pkgs/development/libraries/gstreamer/default.nix

```diff
@@ -1,4 +1,4 @@
-{ callPackage, AudioToolbox, AVFoundation, Cocoa, CoreFoundation, CoreMedia, CoreServices, CoreVideo, DiskArbitration, Foundation, IOKit, MediaToolbox, OpenGL, VideoToolbox }:
+{ callPackage, AudioToolbox, AVFoundation, Cocoa, CoreFoundation, CoreMedia, CoreServices, CoreVideo, DiskArbitration, Foundation, IOKit, MediaToolbox, OpenGL, VideoToolbox, ipu6ep-camera-hal }:
```

Comment:
Please reformat this to one element per line, this line is otherwise getting overly long.

Reply:
done